charts/manila-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -59,6 +59,21 @@ spec:
               mountPath: /var/lib/kubelet/plugins/{{ printf "%s.%s" .protocolSelector $.Values.driverName | lower }}
           resources:
 {{ toYaml $.Values.controllerplugin.snapshotter.resources | indent 12 }}
+        - name: {{ .protocolSelector | lower }}-resizer
+          image: "{{ $.Values.controllerplugin.resizer.image.repository }}:{{ $.Values.controllerplugin.resizer.image.tag }}"
+          args:
+            - "--v=5"
```

Comment:
I think we should avoid using verbose logging by default . Its applicable for all other sidecars as well

Reply:
I checked cinder-csi's manifests and indeed there's no `--v` flag in any of the sidecars. I can fix this in bulk in a separate PR. WDYT?

Reply:
sounds good